searchcore/src/vespa/searchcore/proton/server/documentsubdbcollection.cpp

```diff
@@ -109,10 +111,26 @@ DocumentSubDBCollection::DocumentSubDBCollection(
 
 DocumentSubDBCollection::~DocumentSubDBCollection()
 {
-    for (auto subDb : _subDBs) {
-        delete subDb;
+    size_t numCores = std::max(1u, std::thread::hardware_concurrency());
```

Comment:
I think we should use HwInfo.cpu().cores() here.

Reply:
Fixed